lib/client.js

```diff
@@ -132,6 +135,14 @@ Client.prototype.getJwtFromUrl = function () {
 };
 
 /**
+ * Attempts to replace the JWT in the url with the new JWT passed in the function.
+ * @param {string} newJwt
+ */
+Client.prototype.replaceJWTInUrl = function (newJwt) {
+  window.location.hash = '/?jwt=' + newJwt;
```

Comment:
We don't use the query string for this right now, and I don't think we should be putting the JWT in the query string at all anyways, for the same reasons as social login, so I would consider this a minor issue. We would know if we were changing how stormpath calls into ID Site, anyways.